packages/twenty-front/src/modules/object-record/record-store/states/recordStoreFamilyState.ts

```diff
@@ -2,7 +2,7 @@ import { ObjectRecord } from '@/object-record/types/ObjectRecord';
 import { createFamilyState } from '@/ui/utilities/state/utils/createFamilyState';
 
 export const recordStoreFamilyState = createFamilyState<
-  ObjectRecord | null,
+  ObjectRecord | null | undefined,
```

Comment:
why do we introduce both null and undefined, I feel only one should be here. undefined seems more accurate than null

Reply:
As already discussed, there's not much gained from using null or undefined alone, it creates a complex nullable typing system that complexifies the DX more than it helps.

Reply:
The most common problem is that you'll end up with one system that only handles null, another that only handles undefined, and one day you'll need to have both communicate with each other and you'll have a hard time.